docs/variants/msi_z690/recovery.md

```diff
@@ -26,16 +26,16 @@ here.
   may choose an older one too or in the best case use your firmware backup):
 
 ```bash
-wget https://download.msi.com/bos_exe/mb/7D25v13.zip
-unzip 7D25v13.zip
+wget https://download.msi.com/bos_exe/mb/7D25v1A.zip
+unzip 7D25v1A.zip
 ```
 
 * Flash via external programmer:
 
 > The command line will be different, depending on the programmer you use
 
 ```bash
-flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7D25v13/E7D25IMS.130
+flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7D25v1A/E7D25IMS.130
```

Comment:
The file will not be called E7D25IMS.130 right? I guess it will be `E7D25IMS.1A0`, would have to check the ZIP contents

Reply:
Nice catch, I corrected the error